### PR TITLE
Some adjustments + VC Compat list 

### PR DESCRIPTION
**Pull request by VannyBuns**

### CODE REVIEW

File: cogs/assistance-cmds/vccompatlists.wiiu.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: Wii U inject Compatibility list sites
+help-desc: Lists various links to Wii U VC compatibility sites
+aliases: compatlists
+---
+
+You can see what games are compatibile with the Wii U/WiiVC from these sites.
+
+Wii:
+https://wiki.gbatemp.net/wiki/WiiU_VC_Wii_inject_compatibility_list
```
Comment (Lazr1026) [inline cogs/assistance-cmds/vccompatlists.wiiu.md:10]: The compat lists (minus gamecube) should use the UWUVCI ones. who knows how out of date the gbatemp ones are..

File: cogs/assistance-cmds/vccompatlists.wiiu.md [hunk 2]
```diff
@@ -0,0 +1,34 @@
+---
+title: Wii U inject Compatibility list sites
+help-desc: Lists various links to Wii U VC compatibility sites
+aliases: compatlists
+---
+
+You can see what games are compatibile with the Wii U/WiiVC from these sites.
```
Comment (Lazr1026) [inline cogs/assistance-cmds/vccompatlists.wiiu.md:7]: None of these have vWii VC.
```suggestion
You can see what games are compatibile with the Wii Us Virtual Console from these sites.
```

**Review by Lazr1026 — CHANGES_REQUESTED** [review]

File: cogs/assistance-cmds/vccompatlists.wiiu.md (hunk 2 above)
Comment (VannyBuns) [reply cogs/assistance-cmds/vccompatlists.wiiu.md:7]: Wii and GameCube is what I meant by "WiiVC"

Comment (Lazr1026) [reply cogs/assistance-cmds/vccompatlists.wiiu.md:7]: i think "Wii VC" implies vWii VC, especially with the seperation between "Wii U" and "Wii"

Comment (VannyBuns) [reply cogs/assistance-cmds/vccompatlists.wiiu.md:7]: Right

**Review by lifehackerhansol — APPROVED** [review]

**Pull request merged**